rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/presenter/CovidOtherClinicalDataFragmentPresenterTest.java

```diff
@@ -0,0 +1,49 @@
+package io.ona.rdt.presenter;
+
+import android.app.Activity;
+
+import org.json.JSONException;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.robolectric.util.ReflectionHelpers;
+
+import io.ona.rdt.PowerMockTest;
+import io.ona.rdt.domain.Patient;
+import io.ona.rdt.interactor.CovidOtherClinicalDataFragmentInteractor;
+import timber.log.Timber;
+
+@PrepareForTest(Timber.class)
+public class CovidOtherClinicalDataFragmentPresenterTest extends PowerMockTest {
+
+    private CovidOtherClinicalDataFragmentPresenter covidOtherClinicalDataFragmentPresenter;
+    private CovidOtherClinicalDataFragmentInteractor interactor;
+
+    @Before
+    public void setUp() {
+        covidOtherClinicalDataFragmentPresenter = new CovidOtherClinicalDataFragmentPresenter(null);
+        interactor = Mockito.mock(CovidOtherClinicalDataFragmentInteractor.class);
+        ReflectionHelpers.setField(covidOtherClinicalDataFragmentPresenter, "interactor", interactor);
+    }
+
+    @Test
+    public void testLaunchFormShouldVerifyCorrectFormLaunch() throws JSONException {
+        Mockito.doNothing().when(interactor).launchForm(ArgumentMatchers.any(Activity.class), ArgumentMatchers.anyString(), ArgumentMatchers.any(Patient.class));
+
+        covidOtherClinicalDataFragmentPresenter.launchForm(null, null, null);
```

Comment:
Why not use verifiable arguments here instead of null?

Reply:
fixed.